fronts-client/src/components/Row.tsx

```diff
@@ -1,8 +1,16 @@
 import { styled } from 'constants/theme';
 
-const Row = styled.div<{ flexDirection?: string; gutter?: number }>`
+const Row = styled.div<{
+	flexDirection?: string;
+	gutter?: number;
+	rowGap?: number;
+}>`
 	display: flex;
 	margin: 0 ${({ gutter = 10 }: { gutter?: number }) => `${-(gutter / 2)}px`};
+	flex-wrap: wrap;
+	// row-gap used when wrapped
+	row-gap: ${({ rowGap }: { rowGap?: number }) =>
+		rowGap ? `${rowGap}px` : '0'};
```

Comment:
This adds a more semantic vertical gap between the Trail Image and Select Media sections when the row is wrapped (Select Media beneath Trail Image).